docker/bin/start-api

```diff
@@ -5,15 +5,22 @@ set -o nounset
 
 
 readonly GUNICORN='/venv/bin/gunicorn'
-readonly GUNICORN_WORKERS=${GUNICORN_WORKERS:-4}
+readonly GUNICORN_FORWARDED_ALLOW_IPS="${GUNICORN_FORWARDED_ALLOW_IPS:-}"
+readonly GUNICORN_WORKERS="${GUNICORN_WORKERS:-4}"
 
 readonly BIND_HOST='0.0.0.0'
 readonly BIND_PORT=${GUNICORN_PORT:-8000}
 readonly APP_MODULE='pulpcore.app.wsgi:application'
 
 
-exec "${GUNICORN}" \
-  --bind "${BIND_HOST}:${BIND_PORT}" \
-  --workers "${GUNICORN_WORKERS}" \
-  --access-logfile - \
-  "${APP_MODULE}"
+GUNICORN_OPTIONS=(
+  --bind "${BIND_HOST}:${BIND_PORT}"
+  --workers "${GUNICORN_WORKERS}"
+  --access-logfile -
+)
+
+if [[ -n "$GUNICORN_FORWARDED_ALLOW_IPS" ]]; then
```

Comment:
(minor) "${GUNICORN_FORWARDED_ALLOW_IPS}" for consistency with ${} use elsewhere